FtcCommon/src/main/java/com/qualcomm/ftccommon/AboutActivity.java

```diff
@@ -237,22 +237,15 @@ protected String getLocalIpAddressesAsString() {
   }
 
   /** https://code.google.com/p/android/issues/detail?id=220039 */
+  //Modified for OpenFTC
   protected String getBuildTime() {
-
     String buildTime = "unavailable";
     try {
-      ApplicationInfo ai = getPackageManager().getApplicationInfo(getPackageName(), 0);
-
-      ZipFile zf = new ZipFile(ai.sourceDir);
-      ZipEntry ze = zf.getEntry("classes.dex");
-      zf.close();
-
-      long time = ze.getTime();
-      buildTime = SimpleDateFormat.getInstance().format(new java.util.Date(time));
+      long time = AboutActivity.this.getPackageManager().getPackageInfo(AboutActivity.this.getPackageName(), 0).lastUpdateTime;
```

Comment:
Why did you use `AboutActivity.this.getPackageName()` instead of just `getPackageName()`? Shouldn't they be identical?

Reply:
I concur here